src/libraries/System.Xml.XDocument/ref/System.Xml.XDocument.cs

```diff
@@ -77,33 +77,46 @@ public partial class XAttribute : System.Xml.Linq.XObject
         [System.CLSCompliantAttribute(false)]
         public static explicit operator long (System.Xml.Linq.XAttribute attribute) { throw null; }
         [System.CLSCompliantAttribute(false)]
-        public static explicit operator bool? (System.Xml.Linq.XAttribute attribute) { throw null; }
+        [return: System.Diagnostics.CodeAnalysis.NotNullIfNotNullAttribute("attribute")]
+        public static explicit operator bool? (System.Xml.Linq.XAttribute? attribute) { throw null; }
         [System.CLSCompliantAttribute(false)]
-        public static explicit operator System.DateTimeOffset? (System.Xml.Linq.XAttribute attribute) { throw null; }
+        [return: System.Diagnostics.CodeAnalysis.NotNullIfNotNullAttribute("attribute")]
+        public static explicit operator System.DateTimeOffset? (System.Xml.Linq.XAttribute? attribute) { throw null; }
         [System.CLSCompliantAttribute(false)]
-        public static explicit operator System.DateTime? (System.Xml.Linq.XAttribute attribute) { throw null; }
+        [return: System.Diagnostics.CodeAnalysis.NotNullIfNotNullAttribute("attribute")]
+        public static explicit operator System.DateTime? (System.Xml.Linq.XAttribute? attribute) { throw null; }
         [System.CLSCompliantAttribute(false)]
-        public static explicit operator decimal? (System.Xml.Linq.XAttribute attribute) { throw null; }
+        [return: System.Diagnostics.CodeAnalysis.NotNullIfNotNullAttribute("attribute")]
+        public static explicit operator decimal? (System.Xml.Linq.XAttribute? attribute) { throw null; }
         [System.CLSCompliantAttribute(false)]
-        public static explicit operator double? (System.Xml.Linq.XAttribute attribute) { throw null; }
+        [return: System.Diagnostics.CodeAnalysis.NotNullIfNotNullAttribute("attribute")]
+        public static explicit operator double? (System.Xml.Linq.XAttribute? attribute) { throw null; }
         [System.CLSCompliantAttribute(false)]
-        public static explicit operator System.Guid? (System.Xml.Linq.XAttribute attribute) { throw null; }
+        [return: System.Diagnostics.CodeAnalysis.NotNullIfNotNullAttribute("attribute")]
+        public static explicit operator System.Guid? (System.Xml.Linq.XAttribute? attribute) { throw null; }
         [System.CLSCompliantAttribute(false)]
+        [return: System.Diagnostics.CodeAnalysis.NotNullIfNotNullAttribute("attribute")]
         public static explicit operator int? (System.Xml.Linq.XAttribute attribute) { throw null; }
         [System.CLSCompliantAttribute(false)]
-        public static explicit operator long? (System.Xml.Linq.XAttribute attribute) { throw null; }
+        [return: System.Diagnostics.CodeAnalysis.NotNullIfNotNullAttribute("attribute")]
```

Comment:
`public static explicit operator int? (System.Xml.Linq.XAttribute attribute) { throw null; }` should likely take nullable

Reply:
same foe ` public static explicit operator float (System.Xml.Linq.XAttribute attribute) { throw null; }`

Reply:
and `public static explicit operator System.TimeSpan (System.Xml.Linq.XAttribute attribute)`

Reply:
`public static explicit operator uint (System.Xml.Linq.XAttribute attribute)`
`public static explicit operator ulong (System.Xml.Linq.XAttribute attribute)`
as well

Reply:
It's only in the case of `int?` that is missing nullability in the argument.

Reply:
Good catch, though.